packages/medusa/src/services/order.ts

```diff
@@ -1779,10 +1779,31 @@ class OrderService extends TransactionBaseService {
     const calculationContext = await this.totalsService_.getCalculationContext(
       order
     )
-    const orderItems = [...(order.items ?? [])]
+
+    const { returnable_items } = totalsFieldsOrContext?.includes ?? {}
+
+    const returnableItems: LineItem[] | undefined = returnable_items
+      ? []
+      : undefined
+
+    const isReturnableItem = (item) =>
+      returnable_items &&
+      (item.returned_quantity ?? 0) < (item.fulfilled_quantity ?? 0)
```

Comment:
todo: I believe we should make `item.shipped_quantity` instead, as you can cancel a fulfillment from admin instead of returning it.